src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.cpp

```diff
@@ -1,7 +1,7 @@
 /*
  ISC License
 
- Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+ Copyright (c) 2024, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
Don't update the copyright year in open source code. This attempts rewrite history.  Please leave the original copyright year.  If you want to highlight that new code is added in 2024, you can add a 2nd copyright line.  But, I don't recommend this in these situations.  If some file is copyrighted in 2023, this carries forward to future years without having to be edited.

Reply:
Done